+++ src/Makefile.am	2023-10-13 08:22:51.362607863 +0200
@@ -15,7 +15,7 @@
 
 DISTCHECK_CONFIGURE_FLAGS = --with-completiondir=/tmp
 
-SUBDIRS = etc src win32 doc MacOSX tests
+SUBDIRS = src
 
 dist_noinst_SCRIPTS = bootstrap bootstrap.ci
 dist_noinst_DATA = README \
diff -N -u -r '--exclude=.git' '--exclude=*~' src/src/Makefile.am src/src/Makefile.am
--- src/src/Makefile.am	2023-10-13 08:16:24.471973536 +0200
+++ src/src/Makefile.am	2023-10-13 08:23:42.509071542 +0200
@@ -2,12 +2,8 @@
 EXTRA_DIST = Makefile.mak
 
 # Order IS important
-SUBDIRS = common scconf ui pkcs15init sm \
-		  libopensc pkcs11 tools minidriver
-
-if ENABLE_TESTS
-SUBDIRS += tests
-endif
+SUBDIRS = common scconf ui pkcs15init sm \
+		  libopensc pkcs11 tools
 
 if ENABLE_SM
 SUBDIRS += smm
diff -N -u -r '--exclude=.git' '--exclude=*~' src/src/tools/Makefile.am src/src/tools/Makefile.am
--- src/src/tools/Makefile.am	2023-10-13 08:16:24.535976619 +0200
+++ src/src/tools/Makefile.am	2023-10-13 08:25:48.347133160 +0200
@@ -28,9 +28,7 @@
 	egk-tool-cmdline.h goid-tool-cmdline.h npa-tool-cmdline.h \
 	opensc-asn1-cmdline.h opensc-notify-cmdline.h pkcs11-register-cmdline.h \
 	openpgp-tool-helpers.h
-bin_PROGRAMS = opensc-tool opensc-explorer opensc-asn1 \
-	pkcs15-tool pkcs15-crypt pkcs11-tool pkcs11-register \
-	cardos-tool eidenv openpgp-tool iasecc-tool egk-tool goid-tool
+bin_PROGRAMS = pkcs11-tool
 if ENABLE_OPENSSL
 bin_PROGRAMS += cryptoflex-tool pkcs15-init netkey-tool piv-tool \
 	westcos-tool sc-hsm-tool dnie-tool gids-tool
+++ src/src/tools/Makefile.am	2023-10-13 09:49:12.956198948 +0200
@@ -48,6 +48,7 @@
 # compile with $(PTHREAD_CFLAGS) to allow debugging with gdb
 AM_CFLAGS = $(OPTIONAL_OPENSSL_CFLAGS) $(OPTIONAL_READLINE_CFLAGS)
 AM_CPPFLAGS = -I$(top_srcdir)/src -D'DEFAULT_PKCS11_PROVIDER="$(DEFAULT_PKCS11_PROVIDER)"' -D'DEFAULT_ONEPIN_PKCS11_PROVIDER="$(DEFAULT_ONEPIN_PKCS11_PROVIDER)"'
+LDFLAGS = $(LDLIBS)
 LIBS = \
 	$(top_builddir)/src/libopensc/libopensc.la \
 	$(top_builddir)/src/common/libscdl.la \
